src/app/wallet/overview/widgets/coldstake/coldstake.service.ts

```diff
@@ -72,8 +72,10 @@ export class ColdstakeService implements OnDestroy {
 
       if ('enabled' in coldstakinginfo) {
         this._rpcState.set('ui:coldstaking', coldstakinginfo['enabled']);
+        this.coldStakingEnabled = true;
```

Comment:
Hmm, I think enabled can be 'false' too.

Reply:
ops, true that

Reply:
@claudiocabral fixed